addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/RetrievedFromCacheScanner.java

```diff
@@ -96,27 +95,15 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
                                     logger.debug(
                                             msg.getRequestHeader().getURI().getURI()
                                                     + " was served from a cache, due to presence of a 'HIT' in the 'X-Cache' response header");
-                                Alert alert =
-                                        new Alert(
-                                                getPluginId(),
-                                                Alert.RISK_INFO,
-                                                Alert.CONFIDENCE_MEDIUM,
-                                                Constant.messages.getString(
-                                                        MESSAGE_PREFIX + "name"));
                                 // could be from HTTP/1.0 or HTTP/1.1. We don't know which.
-                                alert.setDetail(
-                                        getDescription(), // Description
-                                        msg.getRequestHeader().getURI().toString(), // URI
-                                        "", // Param
-                                        "", // Attack
-                                        "", // Other info:
-                                        getSolution(), // Solution
-                                        getReference(), // References
-                                        evidence, // Evidence
-                                        0, // 0 - not necessarily an issue at all
-                                        0, // 0 - not necessarily an issue at all
-                                        msg); // HttpMessage
-                                parent.raiseAlert(id, alert);
+                                newAlert()
+                                        .setRisk(Alert.RISK_INFO)
```

Comment:
I didn't set the name as the original could have leveraged `getName()` instead of using i18n a second time.

Reply:
Same below